chains/orchestrator-paras/node/src/rpc.rs

```diff
@@ -58,6 +61,8 @@ pub struct FullDeps<C, P> {
     pub xcm_senders: Option<(flume::Sender<Vec<u8>>, flume::Sender<(ParaId, Vec<u8>)>)>,
     /// Channels for manually activating the randomness
     pub randomness_sender: Option<flume::Sender<(bool, Option<[u8; 32]>)>>,
+    /// Channels for manually excluding container chains from producing blocks
+    pub container_chain_exclusion_sender: Option<flume::Sender<Vec<ParaId>>>,
```

Comment:
let's create an issue for adding this also for dancelight

Reply:
Yeah I can add it in the chain inactivity PR too!